geoportailv3/templates/index.html

```diff
@@ -32,16 +32,16 @@
           <li class="lang icon hidden-xs" ng-class="mainCtrl.languageOpen ? 'active': ''">
             <a href translate ngeo-btn ng-model="mainCtrl.languageOpen">lang</a>
             <ul class="nav toolbox-panel toolbox-panel-top">
-              <li translate ng-class="mainCtrl.lang == 'lb' ? 'active' : ''">
```

Comment:
While you're at it, I think `"{active: mainCtrl.lang == 'lb'}"` offers a nicer syntax in this case.